ui.frontend/src/constants/searchParams.ts

```diff
@@ -0,0 +1,7 @@
+export const searchParams = {
```

Comment:
too generic naming ; it should be scoped to execution;

IMO it could be paired with backend;  `ExecutionQueryParams`

https://github.com/wttech/acm/blob/main/core/src/main/java/com/wttech/aem/acm/core/code/ExecutionQuery.java#L29
?

Reply:
I used to put backend-related things to api.types.ts ;)

Reply:
yeah, i was wondering if it's right place, will move it

Reply:
Done in: [7e25955](https://github.com/wttech/acm/pull/31/commits/7e25955dbe0865100e2a787d4e167bebb855b9f6)